app/routes/questions/$question.tsx

```diff
@@ -110,8 +124,8 @@ export function Question({
   }
 
   return (
-    <article className={cls} {...props}>
-      <h2 onClick={handleToggle} title={isExpanded ? 'Hide answer' : 'Show answer'}>
+    <article className={cls}>
+      <h2 onClick={handleToggle} title={isExpanded ? 'Hide answer' : 'Show answer'} {...dragProps}>
```

Comment:
currently the whole answer is draggable, which in turn means that you can't select the contents to e.g. copy a paragraph. This will make only the header draggable